client/components/GameListEntry.jsx

```diff
@@ -1,11 +1,16 @@
 import React from 'react';
 
 const GameListEntry = (props) => (
-  <div className="game-entry">
-    <div>{props.game.title}</div>
-    <div>{props.game.description}</div>
-    <div>{props.game.price}</div>
-    <div>{props.game.rating}</div>
+  <div className="game-entry" 
+  onMouseEnter={props.handleGameMouseEntry}
```

Comment:
Here on lines 5 and 6 you need to indent your code 4 spaces in total to line up correctly and maintain readability.